src/uploadStrategy/streamChunkStrategy.ts

```diff
@@ -30,22 +30,23 @@ export class StreamChunkStrategy implements UploadStrategy {
     return uploadContentToFileContainer(chunk, batch);
   }
 
-  public postUpload() {
-    return this.closeOpenedStream();
+  public async preUpload() {
+    await this.openStream();
```

Comment:
Has no effect really... but there is no point to return an output in the `preUpload` and `postUpload` phases.